discord/ext/commands/core.py

```diff
@@ -657,6 +660,10 @@ def clean_params(self) -> Dict[str, inspect.Parameter]:
         """
         return self.params.copy()
 
+    @property
+    def cooldown(self) -> Optional[Cooldown]:
+        return self._buckets._cooldown
```

Comment:
Documentation should be moved here.

It should also have `.. versionadded:: 2.0`.

Reply:
Done, sorry for the delay.